tests/phpunit/tests/actions.php

```diff
@@ -348,6 +397,8 @@ function action_that_causes_recursion( $tag ) {
 	/**
 	 * @ticket 9968
 	 * @ticket 17817
+	 *
+	 * @covers ::add_action
```

Comment:
```suggestion
	 * @covers ::remove_action
	 * @covers ::add_action
```

This test is testing the callback registration (add) and deregistration (remove) process. Reviewing the tickets attached it, it's checking for a specific use case of when the callback registration is dynamically changed when the event (tag) is being invoked.